editoast/src/views/timetable.rs

```diff
@@ -558,31 +548,89 @@ async fn conflicts(
     let conflicts = conflict_detection_response.conflicts;
     let conflicts_response: Result<Vec<Conflict>> = conflicts
         .into_iter()
-        .map(|conflict| Conflict::into_conflict_response(conflict, map_string_to_id.clone()))
+        .map(Conflict::from_core_response)
         .collect();
     Ok(Json(conflicts_response?))
 }
 
+async fn retrieve_trains_and_paced_trains(
```

Comment:
`retrieve_trains_and_paced_trains` and `retrieve_simulations` are somewhat similar, but one takes a `DbConnectionPoolV2` and the other one takes a `DbConnection`. Maybe it's a good time to make them both take a `DbConnection`?

Reply:
thanks, done!